apps/gitness/src/pages/repo/repo-create-page.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react'
+import { SandboxRepoCreatePage, CreateRepoProps } from '@harnessio/playground'
+import Header from '../../components/Header'
+import { useGetSpaceURLParam } from '../../framework/hooks/useGetSpaceParam'
+import { useNavigate } from 'react-router-dom'
+
+import {
+  useCreateRepositoryMutation,
+  OpenapiCreateRepositoryRequest,
+  CreateRepositoryOkResponse,
+  CreateRepositoryErrorResponse
+} from '@harnessio/code-service-client'
+
+export const CreateRepo = () => {
+  const createRepositoryMutation = useCreateRepositoryMutation({})
+  const spaceId = useGetSpaceURLParam()
+  const navigate = useNavigate()
+
+  const [apiError, setApiError] = useState<string | null>(null)
+
+  const handleFormSubmit = (data: CreateRepoProps) => {
+    const repositoryRequest: OpenapiCreateRepositoryRequest = {
+      default_branch: 'main',
```

Comment:
Can we add the actual default branch here? It could be `main` or `master` or anything else.